src/modules/modals/TransferTable.vue

```diff
@@ -2,49 +2,56 @@
 	<NcModal v-if="showModal"
 		size="normal"
 		@close="actionCancel">
-		<div class="modal__content">
+		<div class="modal__content" data-cy="transferTableModal">
 			<div class="row">
 				<div class="col-4">
-					<h2>{{ t('tables', 'Transfer tables ownership') }}</h2>
+					<h2>{{ t('tables', 'Transfer table') }}</h2>
 				</div>
 			</div>
 			<div class="row">
-				// TODO: add actual owner
+				<div class="col-4 mandatory space-T">
+					{{ t('tables', 'Owner') }}
+				</div>
+
+				<!-- We don't need to show this, since the current user will only see this if they're the owner? -->
+				<div v-if="localTable !== undefined" class="col-3 inline space-T-small">
+					<NcUserBubble
+						:margin="4"
+						:size="30"
+						:display-name="localTable.ownerDisplayName"
+						:user="localTable.owner" />
+				</div>
 			</div>
```

Comment:
I don't think we should show the current owner on this modal. We already show this on the main "Edit Table" modal itself. No reason having it here, imo